### PR TITLE
Ko | TR

### PR DESCRIPTION
**Pull request by schade61**
# SCP:SL Translation PR Template

## Language Name
ko

## State Type of change (New language, changed existing Etc.)
TR

# Checklist:
- [ ] I have tested the translation changes ingame
- [ ] I have made sure to not commit changes to multiple translations at the same time, or non-translation repo files
- [ ] I have ran `EncodingNormalizer.exe` on my changed files before commiting 
   -  (Exception to this is when edits has __only__ been made in-browser. If this is the case, also check this box.)
- [ ] I have made sure that I'm **not** submitting a duplicate translation 
- [ ] I have made sure to **not** included any advertisements/pointers to non-game stuff other than the translation author in the credits file
- [ ] I have made sure to label my translation folder by [Microsoft Language Tag Standards](https://docs.microsoft.com/en-us/openspecs/windows_protocols/ms-lcid/a9eac961-e77d-41a6-90a5-ce1a8b0cdb9c)
   - Only in cases of **new** translations (Can be tested [Here](https://rextester.com/WDXPS97501)). Also check this box if the change is **not** a new translation
> For more information regarding game translations, please visit our [README](https://github.com/northwood-studios/SCPSL-Translations/blob/master/README.md) page.

### CODE REVIEW

**Comment by schade61** [comment]
What does Line 89 'Flash Taskbar on Spawn' and Line 92 'Bloom' in [NewMainMenu.txt](https://github.com/northwood-studios/SCPSL-Translations/blob/master/Translations/English%20(default)/NewMainMenu.txt) does?

**Comment by ArithFeather** [comment]
> What does Line 89 'Flash Taskbar on Spawn' and Line 92 'Bloom' in [NewMainMenu.txt](https://github.com/northwood-studios/SCPSL-Translations/blob/master/Translations/English%20(default)/NewMainMenu.txt) do?

Flash Taskbar On Spawn - Settings > Other > Toggle button. When a player first spawns or respawns from being dead, and you are not tabbed into the game, it will flash your taskbar icon.

Bloom - Settings > Graphic > Toggle button.  Enable and Disable the Post Processing Bloom. Bloom is like a glow effect for lights and emission.

**Comment by schade61** [comment]
Thank you for reply :)

File: Translations/ko/manifest.json [hunk 1]
```diff
@@ -3,7 +3,7 @@
   "Authors": [
     "Dr-Devan",
     "Arizen",
-    "OneKoin"
+    "schade"
```
Comment (Rnen) [inline Translations/ko/manifest.json:6]: Any reason you replaced this name?

**Comment by Rnen** [comment]
image
Just a FYI

File: Translations/ko/manifest.json (hunk 1 above)
Comment (schade61) [reply Translations/ko/manifest.json:6]: cause its same person

Comment (Rnen) [reply Translations/ko/manifest.json:6]: Alright :smile: Just had to check

**Comment by schade61** [comment]
> image
> Just a FYI

I will work on those files soon. Thank you for notice.

**Comment by Rnen** [comment]
> I will work on those files soon. Thank you for notice.
Will you include them in this PR or a later one? :slightly_smiling_face:

**Comment by schade61** [comment]
> Will you include them in this PR or a later one? :slightly_smiling_face: 

I might update those in this PR.
BTW, what does Line 8 "DROP" means in [InventoryGUI](https://github.com/northwood-studios/SCPSL-Translations/blob/master/Translations/English%20(default)/InventoryGUI.txt)?

**Comment by Rnen** [comment]
> BTW, what does Line 8 "DROP" means in [InventoryGUI](https://github.com/northwood-studios/SCPSL-Translations/blob/master/Translations/English%20(default)/InventoryGUI.txt)?

Asked our programmers, will update with reply.

**Comment by Rnen** [comment]
image

**Comment by schade61** [comment]
> image
> 

Thanks :) Guess I'll have to change my translation.

**Review by Rnen — APPROVED** [review]

**Pull request merged**